modules/core/src/system.cpp

```diff
@@ -554,7 +554,9 @@ struct HWFeatures
         have[CV_CPU_FP16] = true;
     #endif
     #endif
-
+    #if (defined(_WIN32_WCE) && _WIN32_WCE >= 0x800)
+        have[CV_CPU_NEON] = true;
```

Comment:
Perhaps this should be under `defined __arm__` or `_M_ARM` condition.

Reply:
Hi

The WINCE toolchain defines `/D_WIN32_WCE=0x800 /DUNDER_CE /DWINCE /DARM /D_ARM_ /DARMV7 /D_WINDOWS`.  I could add

```
#if defined __arm__ || defined _ARM_
#if defined _WIN32_WCE && _WIN32_WINCE >= 0x800
...
#elif __linux__
...
#elif
...
#endif
```

But feelt much more convoluted. Perhaps 

```
#if defined _ARM__ && defined _WIN32_WCE && _WIN32_WINCE >= 0x800
...
#endif
```
would suffice?

Reply:
as you wish

Reply:
Ok. I went with the _ARM_ solution